Codeinwp/themeisle-companion
# fix: use new X icon

### Summary


- Updated the Socicon to the latest version.
- Added new Twitter Logo as SVG and necessary CSS to keep the formating.

### Will affect visual aspect of the product

YES

### Screenshots 

#### Before

<img width="2243" alt="Screenshot 2024-03-25 at 17 24 24" src="https://github.com/Codeinwp/themeisle-companion/assets/17597852/0c4d17e4-dcf8-4472-9bad-1d74d8222995">

#### After

<img width="2253" alt="Screenshot 2024-03-25 at 17 23 39" src="https://github.com/Codeinwp/themeisle-companion/assets/17597852/af8605e5-ee6d-44f2-a027-64ea8fd452b7">

![image](https://github.com/Codeinwp/themeisle-companion/assets/17597852/bb8dd391-14cd-447b-b7a8-7d40d970e994)


### Test instructions


- In the Social Sharing Module activate the Twitter logo.
- Check the page on both desktop and mobile.

## Check before Pull Request is ready:



Closes #836
